pkg/stis/calstis/cs1/dodqi.c

```diff
@@ -652,6 +653,8 @@ int doppmin, doppmax  i: offsets for Doppler shift
 	    xlength = 2;
 	    ylength = tabrow->length * 2;
 	}
+	else
+	    assert(0);
```

Comment:
We also have to consider that most of the pipeline processing happen as a blackbox on the DP side. When something fails, the first thing they tend to look at are the trailer files. Just an abort message to screen is not going to help them.